docker-compose.yml

```
@@ -7,6 +7,7 @@ services:
       target: development
     volumes:
       - .:/usr/src/app
+      - /usr/src/app/node_modules
     # Override the Dockerfile command
     command: npm run start:dev
```

O que acha de fazer assim?

```suggestion
    volumes:
      - .:/usr/src/app
    # Override the Dockerfile command
    command: bash -c "npm ci && npm run start:dev"
```

---

Pode ser também pow, também pensei nessa possibilidade, deixei a do volume, pq vi uma galera do node usando 😅